settings_test.py

```diff
@@ -48,7 +48,7 @@ def _polite_tmpdir():
 # is just too annoying for tests, so disable it.
 CACHE_COUNT_TIMEOUT = -1
 CELERY_ROUTES = {}
-CELERY_ALWAYS_EAGER = True
+CELERY_ALWAYS_EAGER = False
```

Comment:
I half hoped that there was some magic to automatically do this in tests with the latest version :)